src/System.Runtime.Extensions/src/System/IO/BufferedStream.cs

```diff
@@ -1246,6 +1246,30 @@ public override void SetLength(long value)
             _stream.SetLength(value);
         }
 
+        public override void CopyTo(Stream destination, int bufferSize)
```

Comment:
Never mind, https://github.com/dotnet/corefx/pull/9929/files was for `Stream.CopyTo`, not `BufferedStream.CopyTo`.